common/grouping/grouping-en.html

```diff
@@ -0,0 +1,252 @@
+---
+{
+	"title": "Grouping Content - Scaffolding",
+	"language": "en",
+	"altLangPage": "grouping-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-06-28"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<div class="alert alert-warning">
+<p>The purpose of this page is to test all native content grouping related elements, if they are aligned with our design and are compliant to our accessibility guideline when used as is without any special customization. The following include all content grouping elements in the HTML5 specification and a few examples was inspired by the <a href="https://html.spec.whatwg.org/multipage/forms.html#forms">WHATWG section 4.4</a> as February 2023. This page may not contain all the possible grouping content element combination.</p>
+</div>
+
+<h2>The <code>p</code> element</h2>
+<p></p>
+<pre><code>&lt;p>&lt;/p></code></pre>
+
+<h2>The <code>hr</code> element</h2>
+<hr>
+<pre><code>&lt;hr></code></pre>
+
+
+<h2>The <code>pre</code> element</h2>
+<pre> </pre>
+<pre><code>
+&lt;pre>&lt;code>&lt;p> This is a pre element example&lt;/p>&lt;/code>&lt;/pre>
+</code></pre>
+
+<h2>The <code>blockquote</code> element</h2>
+<blockquote>
+	<p>This is a blockquote example</p>
+</blockquote>
+<pre><code>&lt;blockquote&gt;
+&lt;p&gt; ... &lt;/p&gt;
+&lt;/blockquote&gt;</code></pre>
+
+<h2>The <code>ol</code> element</h2>
+<p>I have lived in the following countries </p>
```

Comment:
Remove the paragraph as it don't add value to the purpose of demoing the `<ol>` element
```suggestion
```